docs/user/exports.rst

```diff
@@ -0,0 +1,90 @@
+The creation is done asynchronously. You will receive a notification once the archive is ready to download. You can then download the archive by clicking |download_archive| on the course management overview page or the exam checklist page.
+
+Export programming exercise material
+------------------------------------
+Export the exercise material (template,solution,test, auxiliary repositories and problem statement and other relevant information such as points) of a programming exercise.
+To export the material click the |download_exercise| button on the exercise details page.
+
+
+Export programming exercise student repositories
+------------------------------------------------
+Export the student repositories (this can include the repositories for both graded and practice participations) of a programming exercise
+To export the repositories click the |export| button and then the |download_repos| button on the |download_scores| page.
+
+Export exercise results
+-----------------------------------
+Export the results of a specific exercise. This is supported for all exercise types.
+To export the results click the |export| button and then the |export_results| button on the |download_scores| page.
+
+Export quiz exercise
+--------------------
+Exports the questions and the sample solution of a quiz in JSON format
+You can export a a quiz exercise by clicking the |export_quiz| button on the exercises overview page.
+
+Export exercise submissions
+---------------------------
+Export the submissions of all students that participated in a specific exercise. This is supported for text, modeling and file upload exercises.
+Text submissions are exported as a zip file containing all submissions as text files.
+Modeling submissions are exported as a zip file containing all submissions as json files.
+File upload submissions are exported as a zip file containing all submitted files.
+To export the submissions click the |export_scores| button and then the |export_submissions| button on the |download_scores| page.
+
+Export course/exam scores
+-------------------------
+Export the scores of all students that participated in a specific course or exam. This is supported for all exercise types.
+The scores are exported in CSV format.
+To export the scores of a course click on the |scores_navigation_bar| tab in the course management navigation bar and then the |export_scores| button.
+For exams you can export the scores by clicking the |download_scores| button on the exam checklist page and then |export_scores| button.
+
+Export user data
+----------------
+Export all data Artemis stores about a specific user. This includes information such as name or email, exercise submissions, results, feedbacks the user received, messages they've sent.
+You can request a data export by clicking |privacy_statement| and |data_export|. Once the export has been created you will receive an email with a download link.
```

Comment:
```suggestion
You can request a data export by clicking |privacy_statement| in the page footer and |data_export|. Once the export has been created you will receive an email with a download link.
```
It might not be obvious to users where this link is.